01-JS-DOM-APIs/exercise1/js/main.js

```diff
@@ -0,0 +1,9 @@
+function fadein(section) {
+  if (section.className == "section section_transparency_full") {
+    section.className = "section section_transparency_fadein";
+  }
+}
+
+let helloWorldSection = document.getElementById("hello-world");
+
+document.addEventListener("load", fadein(helloWorldSection));
```

Comment:
`addEventListener` second parameter must be a callback, here you are directly calling the function fadein, so it will receive `undefined` as second parameter (because fadein is returning nothing). In this case, you will need to bind the function with the section argument.